Documentation/advanced-configuration.md

```diff
@@ -615,3 +616,27 @@ To recheck that the Phantom OSD got removed, re-run the following command and ch
 ```bash
 ceph osd tree
 ```
+
+## Change Failure Domain
+Since `v0.9` it is possible to indicate to Rook how the default CRUSH failure domain rule must be configured in order to ensure that replicas or erasure code shards are separated across hosts and a single host failure will not affect availability. For instance, in the case of a block pool:
```

Comment:
I'd suggest not mentioning the version since we have versioned docs. And was this really only available since 0.9? It seems like it's been there since 0.8, but time is a blur.

Reply:
Updated